docs/modules/ROOT/pages/micrometer-registry-newrelic-telemetry.adoc

```diff
@@ -0,0 +1,19 @@
+= New Relic (End of Life)
+
```

Comment:
Is this backwards? You have the end of life notification in the telemetry adoc?

Reply:
found that I had fixed it but failed to commit it...